ai-service/src/ai_service/main.py

```
@@ -1,112 +1,176 @@
+def answer_question(
+    user_question: str,
+    number_of_results: int,
+    collection_name: str,
+) -> str:
     try:
-        repo_url = "https://github.com/kristifidani/rust_grpc_poc.git"
-        ingest_github_project(repo_url)
-        answer_question("What is this project about? How does it work?")
+        question_embedding = embedder.embed_text(user_question)
+        results = db.query_chunks(
+            question_embedding, number_of_results, collection_name
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)